### PR TITLE
Thermostat Card: Add Step up and Down

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

image

Similar to the Google home version:

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request: [TODO]

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [x] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/cards/hui-thermostat-card.ts [hunk 1]
```diff
@@ -238,6 +240,28 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
             </div>
           </div>
           <div id="info">
+            <div class="step-icons">
+              <ha-icon-button
+                class="minus step-icon ${classMap({
+                  disabled: Array.isArray(this._setTemp) && !this._lastSetMode,
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-thermostat-card.ts:246]: This is weird, we can better remove the buttons if they are only going to work after interaction with the slider

Comment (zsarnett) [reply src/panels/lovelace/cards/hui-thermostat-card.ts:246]: Removing and then adding would either cause the card height to jump or for there to normally be a large gap between the bottom of the slider and the icons

**Comment by zsarnett** [comment]
image

**Comment by zsarnett** [comment]
@bramkragten Should I close this and just redo this card with no slider? or just remove the slider for the Narrow cards?

**Comment by bramkragten** [comment]
I think we should remove the slider from the narrow cards. Having mode buttons and buttons for the temp is more useful than just the slider.

And we should scale the buttons with the slider and align it with the slider, like with the google slider.

**Comment by zsarnett** [comment]
Closing. I think the future is a non-slider card

**Pull request closed without merging**